cmd/boot-script-service/main.go

```diff
@@ -450,7 +464,7 @@ func main() {
 
 	// try and fetch JWKS from issuer
 	if requireAuth {
-		for i := 0; i <= retryAttempts; i++ {
+		for i := 0; i <= int(authRetryCount); i++ {
```

Comment:
I think it would be better to do

```go
for i := uint64(0); i <= authRetryCount; i++ {
```
here so the type of `i` can remain the same as `authRetryCount`.

Reply:
This looks good.